bluechi.spec.in

```diff
@@ -3,6 +3,10 @@
 %global with_python 1
 %endif
 
+# coverage collection is disabled by default , it can be enabled passing `--define "with_coverage 1"` option to rpmbuild
+%if 0%{?with_coverage} 
```

Comment:
You should remove `?`, because it returns 1 if `with_coverage` exists, but not its value:

```
%if 0%{with_coverage}
%global coverage_flags -Dwith_coverage=true
%endif
```

Reply:
My mistake, the correct way should be:

```
%if 0%{?with_coverage}
%global coverage_flags -Dwith_coverage=true
%endif
```